Documentation/tutorial/walkthrough_create_a_docfx_project.md

```diff
@@ -5,16 +5,16 @@ By completing this walkthrough, you'll become familiar with the workflow of `doc
 
 Step1. Setup docfx
 ------------------------
-Download *docfx* from http://dotnet.github.io/docfx/. [Getting Started with docfx](http://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-exe-directly) describes 3 ways to install *docfx*. This walkthrough uses the first one: use *docfx.exe* directly.
+Download *docfx* from http://dotnet.github.io/docfx/. [Getting Started with docfx](http://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-exe-directly) describes three ways to install *docfx*. This walkthrough uses the first one: Use *docfx.exe* directly.
 
 1. Download *docfx.zip* and unzip it to `D:\docfx\`
-2. Add `D:\docfx\` to `PATH` so that command `docfx` and be directly called from everywhere for convenience. E.g. for Windows, `setx PATH "%PATH%;D:\docfx\"`
+2. Add `D:\docfx\` to `PATH` so that command `docfx` and be directly called from everywhere for convenience. (for example, for Windows, `setx PATH "%PATH%;D:\docfx\"`
 
-Step2. Init a docfx project
+Step2. init a docfx project
```

Comment:
I was using the capitalization that would be used in the command. Should it be "Init" and "DocFX" above since they're in a sentence? Your call. @vicancy